CONTRIBUTING.md

```diff
@@ -67,7 +67,8 @@ their personal fork and create pull requests to bring those changes into the
 source repository.
 
 Ensure that your changes conform to how the rest of the crate is written and follows a 
-similar API, patterns, namings & other conventions. 
+similar API, patterns, namings & other conventions. Also ensure that your code is formatted
+using [rustfmt].
```

Comment:
No. There must be a special check in CI.

Reply:
Yah but CI doesn't fix the code,it will only check it, the user still has to run `fmt`

Reply:
OK. I agree.